charts/mattermost-calls-offloader/values.yaml

```diff
@@ -0,0 +1,83 @@
+# Default values for mattermost-calls-offloader.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+image:
+  repository: mattermost/calls-offloader
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.1.0"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: NodePort
+  port: 4545
+  targetPort: 4545
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+config: 
+  - name: K8S_NAMESPACE # Forwarding the namespace to be used for creation of new resources.
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+  - name: LOGGER_ENABLEFILE
+    value: "false"
+  - name: JOBS_APITYPE
+    value: "kubernetes"
+  - name: JOBS_MAXCONCURRENTJOBS
+    value: "1"
+  - name: API_SECURITY_ALLOWSELFREGISTRATION # This should only be set to true if running the service inside a private network.
+    value: "true"
+  - name: LOGGER_CONSOLELEVEL
+    value: "DEBUG"
```

Comment:
I think as a default value we can add INFO here vs DEBUG

Reply:
Thanks fixed